sqldelight-gradle-plugin/src/test/fixtures/generate-view-mappers/expected/com/sample/Test1Model.java

```diff
@@ -53,24 +53,24 @@
   @Nullable
   Long column2();
 
-  interface Other_selectModel {
-    View1Model view1();
+  interface Other_selectModel<T2 extends Test1Model, V0 extends View1Model> {
```

Comment:
they're no longer indexed, the number is the column index of the first column for that view/table. I can just increment the column index if its still better to start at 1